op-node/rollup/superchain.go

```diff
@@ -99,5 +102,9 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 	if superChain.Config.ProtocolVersionsAddr != nil { // Set optional protocol versions address
 		cfg.ProtocolVersionsAddress = common.Address(*superChain.Config.ProtocolVersionsAddr)
 	}
+	if chainID == labsDevnet {
```

Comment:
Why isn't this part of the baked in file? It's part of the rollup.json.

Reply:
rollup.json is not baked into the registry. We didn't add one-off configuration to the registry, to keep chains more uniform.

Reply:
Chains should be kept uniform, however it is a valid per-chain config & I think we should be more explicit in storing the full chain config with each chain rather than having to override it here.

Reply:
See my other comment, there's an argument here that per-chain config of these values should be out of scope of the superchain-registry, since superchains shouldn't have differences in these configs. It's a temporary artifact from pre-superchain world. We can delete this when Goerli is shut down.

If we actually needed to officially support superchain chains in the future with some differences in the rollup config, then I would also suggest to add this to the superchain-registry. But in this case, I'd be fine with as it is. Still think that a code comment about this would be nice here.